wordpress-plugin/gk-block-mcp/tests/Connect/SettingsPageTabsTest.php

```
@@ -409,6 +409,139 @@ public function test_register_settings_registers_the_toggle_options() {
+	 * page, fatal-error protection, recovery-mode email). The fix contains the
+	 * failure inside register_settings(). This pins both halves: the throw does
+	 * not escape, and later `admin_init` callbacks still run.
+	 */
+	public function test_register_settings_failure_does_not_escape_admin_init() {
+		remove_all_actions( 'admin_init' );
+
+		( new Settings_Page( new Block_Inventory() ) )->register();
+
+		add_filter(
+			'register_setting_args',
+			static function ( $args, $defaults, $option_group, $option_name ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$defaults'. (undefined)

(UnusedFormalParameter)